src/sprints/sprints.response.ts

```diff
@@ -222,3 +223,53 @@ export class CheckinSubmissionResponse {
     @ApiProperty({ example: "2023-11-30T06:47:11.694Z" })
     createdAt: Date;
 }
+
+class VoyageNumber {
+    @ApiProperty({ example: 46 })
+    number: number;
+}
+
+class VoyageTeamMemberTeamId {
+    @ApiProperty({ example: 4 })
+    voyageTeamId: number;
+}
+
+class SprintNumberWithVoyageNumber {
+    @ApiProperty({ example: 1 })
+    number: number;
+
+    @ApiProperty({ example: 46 })
+    voyage: VoyageNumber;
+}
+
+export class CheckinFormResponse {
```

Comment:
I think this is not used, so the response is still showing the old ones, 
![image](https://github.com/user-attachments/assets/6ca4cd2d-ea5f-4141-9720-f43987444a1f)
I assume it's supposed to go here
![image](https://github.com/user-attachments/assets/7bb6d326-2143-4188-a5f1-b7f522471d0b)

Reply:
Oh lol, I managed to confuse myself by making the name similar to CheckinSubmissionResponse